[2023-03-20 18:35:18,360] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: to_posgrest_v4.8.read_and_load_temperature manual__2023-03-20T18:35:13.760035+00:00 [queued]>
[2023-03-20 18:35:18,396] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: to_posgrest_v4.8.read_and_load_temperature manual__2023-03-20T18:35:13.760035+00:00 [queued]>
[2023-03-20 18:35:18,397] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-20 18:35:18,397] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-03-20 18:35:18,398] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-20 18:35:18,435] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): read_and_load_temperature> on 2023-03-20 18:35:13.760035+00:00
[2023-03-20 18:35:18,441] {standard_task_runner.py:52} INFO - Started process 21318 to run task
[2023-03-20 18:35:18,449] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'to_posgrest_v4.8', 'read_and_load_temperature', 'manual__2023-03-20T18:35:13.760035+00:00', '--job-id', '175', '--raw', '--subdir', 'DAGS_FOLDER/to_postgresql.py', '--cfg-path', '/tmp/tmp2vw8jqrl', '--error-file', '/tmp/tmpzog3j_t5']
[2023-03-20 18:35:18,451] {standard_task_runner.py:77} INFO - Job 175: Subtask read_and_load_temperature
[2023-03-20 18:35:18,758] {logging_mixin.py:109} INFO - Running <TaskInstance: to_posgrest_v4.8.read_and_load_temperature manual__2023-03-20T18:35:13.760035+00:00 [running]> on host 0b4cdb459521
[2023-03-20 18:35:19,095] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=sakabuana31
AIRFLOW_CTX_DAG_ID=to_posgrest_v4.8
AIRFLOW_CTX_TASK_ID=read_and_load_temperature
AIRFLOW_CTX_EXECUTION_DATE=2023-03-20T18:35:13.760035+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-20T18:35:13.760035+00:00
[2023-03-20 18:35:19,179] {base.py:79} INFO - Using connection to: id: postgres_localhost. Host: host.docker.internal, Port: 5432, Schema: final, Login: ***, Password: ***, extra: {}
[2023-03-20 18:35:19,253] {dbapi.py:225} INFO - Running statement: 
            CREATE TEMPORARY TABLE temp_temperature (
                date DATE PRIMARY KEY,
                min DECIMAL(5, 2),
                max DECIMAL(5, 2),
                normal_min DECIMAL(5, 2),
                normal_max DECIMAL(5, 2),
                mean DECIMAL(5, 2),
                status_temp int
            );

            COPY temp_temperature FROM '/opt/***/data/output-csv/dataclean-temperature-lasvegas.csv' DELIMITER ',' CSV HEADER;
            INSERT INTO temperature (date, min, max, normal_min, normal_max, mean, status_temp)
            SELECT date, min, max, normal_min, normal_max, mean, status_temp
            FROM temp_temperature
            ON CONFLICT (date) DO NOTHING;
        , parameters: None
[2023-03-20 18:35:19,327] {dbapi.py:233} INFO - Rows affected: 0
[2023-03-20 18:35:19,417] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=to_posgrest_v4.8, task_id=read_and_load_temperature, execution_date=20230320T183513, start_date=20230320T183518, end_date=20230320T183519
[2023-03-20 18:35:19,492] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-20 18:35:19,748] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
